coralogix-eventbridge-policy/README.md

```diff
@@ -0,0 +1,10 @@
+# EventBridge Policy
+
+The module will create a policy on a given EventBridge Event Bus so that Coralogix can send data to it.
+
+## Fields
+
+| Parameter          | Description                                                                                   | Default Value | Required |
+|--------------------|-----------------------------------------------------------------------------------------------|---------------|----------|
+| CustomCoralogixArn | In case you want to use a custom coralogix arn enter the aws account id that you want to use. | n\a           |          |
```

Comment:
The name is misleading you dont ask for arn you ask for the account id

Reply:
We copied it from the s3 template https://github.com/coralogix/cloudformation-coralogix-aws/blob/master/coralogix-s3-archive/template.yaml#L22.
Anyway, I'll change it.

Reply:
Updated the name